.github/workflows/dev-build.yml

```diff
@@ -14,8 +14,11 @@ jobs:
       - uses: actions/setup-node@e33196f7422957bea03ed53f6fbb155025ffc7b8 # v3.7.0
         with:
           node-version-file: './package.json'
+      - name: Install root dependencies
+        run: npm ci
+        shell: bash
       - name: Install Dependencies
-        run: npm ci && lerna bootstrap --ignore-scripts -- --legacy-peer-deps
+        run: npx lerna bootstrap --ignore-script -- --legacy-peer-deps
```

Comment:
Non-blocking - can we make sure that there's a sentence or two in the commit message for the merge that mentions this + its motivation?